scripts/gceworker.sh

```diff
@@ -30,12 +30,16 @@ case "${cmd}" in
            --boot-disk-device-name "${NAME}" \
            --scopes "default,cloud-platform"
     gcloud compute firewall-rules create "${NAME}-mosh" --allow udp:60000-61000
+    gcloud compute firewall-rules create "${NAME}-et" --allow tcp:2022
 
     # Retry while vm and sshd to start up.
     retry gcloud compute ssh "${NAME}" --command=true
 
     gcloud compute copy-files "build/bootstrap" "${NAME}:bootstrap"
     gcloud compute ssh "${NAME}" --ssh-flag="-A" --command="./bootstrap/bootstrap-debian.sh"
+    # This was not put in bootstrap-debian.sh because it's for Ubuntu (the Debian version of it
+    # was broken).
```

Comment:
Where is bootstrap-debian used to bootstrap something that's not actually Ubuntu? Maybe we can just rename that thing bootstrap-ubuntu?